packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -547,11 +547,11 @@ export const messages = {
   conflict_detect_empty_results:
     'Conflict Detection failed to retrieve org file state',
   conflict_detect_conflicts_during_deploy:
-    'Conflicts are detected while deploying metadata. Select Override Conflicts to proceed or Cancel to view the conflicts.',
+    'Conflicts were detected while deploying metadata. Choose how to proceed.',
```

Comment:
Confirming that these messages were reviewed by Sonal?

Reply:
Yeah, they were! @sbudhirajadoc Could you approve this pull request to keep track? Thanks, Sonal!

Reply:
Looks good! Thanks!